packages/admin-ui/src/filters/use-filter-status.ts

```diff
@@ -0,0 +1,29 @@
+import { useState, useMemo } from 'react'
+
+import type { ComboboxState } from '../combobox/combobox.state'
+
+import type { AnyObject } from '@vtex/admin-ui-util'
+import type { FilterStatus } from './filter/filter.state'
```

Comment:
This generates a circular dependency, it's better to export this type from `use-filter-status.ts` file